rosetta-source/src/main/rosetta/product-qualification-func.rosetta

```diff
@@ -609,6 +619,13 @@ func Qualify_EquitySwap_ParameterReturnDividend_Basket: <"Qualifies a product as
 		// qualifies that underlier is a security (single name)
 		performancePayout -> underlier -> basket only exists
 
+func Qualify_BaseProduct_EquityForward: <"Qualifies a product as having the Asset Class classification Equity and Base Product Classification Forward.">
+    inputs: economicTerms EconomicTerms (1..1)
+    output: is_product boolean (1..1)
+    set is_product:
+        Qualify_AssetClass_Equity(economicTerms) = True
+        and economicTerms -> payout -> forwardPayout only exists
+
```

Comment:
What is the use case for the addition of this function ?

Reply:
It is part of the composable qualification logic that determines whether the economic terms match the [ISDA Taxonomy](https://www.isda.org/2019/07/24/isda-taxonomy-v1-0/) Base Product definition for Equity Forwards.

Reply:
It is consistent with other composable qualification functions for ISDA Taxonomy Base Product: `Qualify_BaseProduct_EquitySwap`, `Qualify_BaseProduct_IRSwap`, `Qualify_BaseProduct_CrossCurrency`, `Qualify_BaseProduct_Fra` and `Qualify_BaseProduct_Inflation`